src/bitcoin/poller/looper.rs

```diff
@@ -156,6 +156,23 @@ fn update_coins(
     }
 }
 
+// Add missing transactions to the database.
+fn add_missing_txs(bit: &impl BitcoinInterface, db_conn: &mut Box<dyn DatabaseConnection>) {
+    let txids = db_conn.list_missing_txids();
+    log::debug!("Missing txids: {:?}", txids);
+    // Take chunks to avoid storing too many transactions in memory.
+    for chunk in txids.chunks(100) {
```

Comment:
What you're iterating over is already stored in memory, no matter the chunks you take.

Reply:
All the txids are in memory, but using chunks here is in order not to store all txs themselves in memory.

Reply:
Oh my bad i read too quickly when writing the main review comment.